README.md

```diff
@@ -1,2 +1,41 @@
-# go-ipmctl
+# Go language bindings for the ipmctl API for managing PMMs
+
 Go bindings for the libipmctl native C lib to enable management of DCPMMs from Go source 
+
+This is a Go interface for [ipmctl](https://github.com/intel/ipmctl) which is also a work in progress.
```

Comment:
don't think you need "which is also a work in progress"

Reply:
Agreed. Will change.